common/app/views/fragments/footer.scala.html

```diff
@@ -8,7 +8,8 @@
 @import conf.switches.Switches._
 
 <footer class="l-footer u-cf" data-link-name="footer" data-component="footer">
-    <div class="l-footer__primary @if(NewHeader.isSwitchedOn) {hide-on-mobile}">
+    @* TODO: remove this block when we are ready for the new desktop design *@
+    <div class="l-footer__primary hide-on-mobile">
```

Comment:
Removing a opening `<div>` here but I don't see where the related `</div>` is being deleted. Is it just me?